src/Surfnet/StepupRa/RaBundle/Security/Authentication/Provider/SamlProvider.php

```diff
@@ -99,8 +106,11 @@ public function authenticate(TokenInterface $token)
             $roles[] = 'ROLE_RA';
         }
 
+        $institutionConfigurationOptions = $this->institutionConfigurationOptionsService
+            ->getInstitutionConfigurationOptionsFor($identity->institution);
+
```

Comment:
Maybe throw a specific exception (or something like that) if no institution configuration options can be found?